src/services/channel.service.ts

```diff
@@ -55,7 +55,7 @@ export class ChannelService {
     } else if (minimumChannelPermission === ChannelType.Bot) {
```

Comment:
nit: put this change in separate PR so we have a stamp of when this specific, very different and general, thing was changed